src/components/ha-selector/ha-selector-number.ts

```diff
@@ -29,13 +29,17 @@ export class HaNumberSelector extends LitElement {
   protected render() {
     const isBox = this.selector.number.mode === "box";
 
+    const fullLabel = `${this.label}${
```

Comment:
Worth to put into a memoizeOne()?

Reply:
Don't put this in the label, we have the suffix for this